README.rst

```diff
@@ -56,6 +58,12 @@ Usage Example
 
     print(data)
     # [0, 1, 2]
+    
+    iss.i2c.write_ad2(0x1e,0x0002,[0x80, 0x0])
```

Comment:
Good idea to add an example for a 16-bit address. A comment would be useful, to explain that _ad2 means 2-byte address.